src/gam/__init__.py

```diff
@@ -995,6 +998,12 @@ def buildGAPIObject(api):
     return service
 
 
+def buildGAPIObjectNoAuthentication(api):
+    GM_Globals[GM_CURRENT_API_USER] = None
+    http = transport.create_http(cache=GM_Globals[GM_CACHE_DIR])
```

Comment:
httpc or http_client to avoid conflict with the Python http module.

Reply:
I don't understand

Reply:
Oh, now I do